docs/content.md

```diff
@@ -118,7 +118,7 @@ not_found = api.content.get(UID='notfound')
 
 ## Find content objects
 
-You can use the find function to search for content.
+You can use the {meth}`api.content.find` method to search for content.
```

Comment:
It's a function, not a method, since it isn't contained inside a class.
```suggestion
You can use the {meth}`api.content.find` function to search for content.
```

Reply:
This should be:

```
You can use the {function}`api.content.find` function to search for content.
```

I'll tweak it in another PR.

Reply:
Hmm, then maybe a lot of links like this need to be updated in the plone.api docs?

Reply:
I think using the correct Python object in the docs source is helpful to authors, but not so much to readers of HTML. See [Cross-referencing Python objects in the Python domain](https://www.sphinx-doc.org/en/master/usage/domains/python.html#cross-referencing-python-objects) for supported options, some of which are recent additions.

The HTML markup has `py-func` or `py-meth` as a class. However, I wouldn't do anything unique to the inline display for each Python object.

![Screenshot 2025-02-18 at 4 18 52 PM](https://github.com/user-attachments/assets/18d4d946-3bb0-4eba-a82d-dc22f225fb2c)

Could this be a good first-timer task, specifically to review all the Python objects and correct them?